GEOSwift/GeoJSON.swift

```diff
@@ -233,68 +234,62 @@ private func ParseGEOJSONGeometry(_ type: String, coordinatesNSArray: NSArray) -
 
     case "Polygon":
         return GEOJSONCreatePolygonFromRepresentation(coordinatesNSArray)
-        
+
     case "MultiPolygon":
         // For type "MultiPolygon", the "coordinates" member must be an array of Polygon coordinate arrays.
-        var polygons = Array<Polygon>()
+        var polygons = [Polygon]()
         for representation in coordinatesNSArray {
             guard let polyRepresentation = representation as? NSArray,
                   let geom = GEOJSONCreatePolygonFromRepresentation(polyRepresentation) else {
                 return nil
             }
-            
+
             polygons.append(geom)
         }
         return MultiPolygon(polygons: polygons)
-    
+
     default:
         return nil
     }
 }
 
-// MARK:
+// MARK: 
```

Comment:
This is interesting. Looks like swiftlint may have autocorrect rules that are in conflict (trailing whitespace vs MARK: formatting)